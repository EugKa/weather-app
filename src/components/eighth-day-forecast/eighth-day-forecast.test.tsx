import React from 'react';
import { shallow } from "enzyme";
import { EighthDayForecast } from "..";

const props = {
   params: "London",
   data: {
        clouds: 83,
        dew_point: 8.99,
        dt: 1632740400,
        feels_like: {day: 15.08, night: 11.95, eve: 16.3, morn: 16.09},
        humidity: 64,
        moon_phase: 0.7,
        moonrise: 1632774960,
        moonset: 1632747420,
        pop: 1,
        pressure: 1011,
        rain: 3.49,
        sunrise: 1632722081,
        sunset: 1632764890,
        temp: {day: 15.78,
            eve: 17.12,
            max: 17.67,
            min: 12.72,
            morn: 16.17,
            night: 12.72},
        uvi: 2.17,
        weather: [{id: 501, main: 'Rain', description: 'moderate rain', icon: '10d'}],
        0: {id: 501, main: 'Rain', description: 'moderate rain', icon: '10d'},
        length: 1,
        wind_deg: 201,
        wind_gust: 20.34,
        wind_speed: 10.06,
   },
   timezone_offset : 3600
}


const setUp = (props:any) => shallow(<EighthDayForecast {...props}/>)

describe('EighthDayForecast component', () => {    
    it("should render EighthDayForecast component with propsData", () => {
        const component = setUp(props)
        expect(component).toMatchSnapshot()
    }) 
})
