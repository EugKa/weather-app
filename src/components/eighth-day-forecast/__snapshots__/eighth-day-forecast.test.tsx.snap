// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EighthDayForecast component should render EighthDayForecast component with propsData 1`] = `
<div
  className="eighthDayForecast"
>
  <div>
    Mon, Sep 27
  </div>
   
  <div
    className="imgWrapper"
  >
    <ForwardRef(CardMedia)
      component="img"
      height={70}
      image="http://openweathermap.org/img/wn/10d@2x.png"
    />
  </div>
  <div
    className="text"
  >
    12
     / 
    17
    â„ƒ
  </div>
  <div
    className="text"
  >
    Moderate rain
  </div>
</div>
`;
